microsoft/testsuites/performance/storageperf.py

```diff
@@ -125,6 +130,190 @@ def perf_premium_datadisks_io(self, node: Node, environment: Environment) -> Non
             max_iodepth=64,
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test performance of nfs server over TCP with
+        VM's initialized with SRIOV network interface.
+        """,
+        priority=2,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            min_count=2,
+            min_core_count=72,
```

Comment:
What's the reason to limit core count?

Reply:
On a second thought, I think this dependency should not be required